homeassistant/components/tuya/sensor.py

```diff
@@ -476,6 +476,32 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Smart Water Timer
```

Comment:
Is there docs available that can be linked?

Reply:
No, sorry, there isn't docs